AWSQueueManager.php

```diff
@@ -108,17 +108,29 @@ public function getQueues($refresh=false)
     }
 
     /**
+     * Important Note: If the SQS queue is FIFO type we MUST send the MessageGroupId also,
+     * otherwise the request will fail.
+     * @link https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SendMessage.html
+     *
+     * Define if the SQS queue is a FIFO one via passing the `MessageGroupId` on $options param.
+     *
      * @param string $url     url of the queue to send message
      * @param string $message message to send
      * @param array  $options extra options for the message
      * @return boolean message was succesfull
      */
-    public function send($url, $message, $options=array())
+    public function send($url, $message, $options= [])
     {
-        $this->_sqs->sendMessage(array_merge(array(
+        $messageData = [
             'QueueUrl'    => $url,
             'MessageBody' => $message,
-        ),$options));
+        ];
+
+        if (isset($options['MessageGroupId'])) {
+            $messageData['MessageGroupId'] = $options['MessageGroupId'];
```

Comment:
Given that we do `array_merge($messageData,$options)` , do we need to do this copy? 

It might actually work without any changes?

Reply:
No we don't on the single send, fixed and sorry